packages/liveblocks-react/src/__tests__/index.test.tsx

```diff
@@ -0,0 +1,316 @@
+import type { BaseUserMeta, Json, JsonObject } from "@liveblocks/client";
+import type { ServerMsg } from "@liveblocks/client/internal";
+import {
+  ClientMsgCode,
+  CrdtType,
+  ServerMsgCode,
+} from "@liveblocks/client/internal";
+import { rest } from "msw";
+import { setupServer } from "msw/node";
+
+import {
+  useMyPresence,
+  useObject,
+  useOthers,
+  useRoom,
+} from "./_liveblocks.config";
+import { act, renderHook, waitFor } from "./_utils"; // Basically re-exports from @testing-library/react
+
+/**
+ * https://github.com/Luka967/websocket-close-codes
```

Comment:
I would use this reference instead: https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code

Reply:
Ah, thx — can update! 👍